ffyaml/ffyaml_test.go

```diff
@@ -71,3 +71,75 @@ func TestParser(t *testing.T) {
 		})
 	}
 }
+
+func TestParserAllTypesEmptyVals(t *testing.T) {
```

Comment:
Why does this need to be a separate test, rather than part of TestParser?

Reply:
> Why does this need to be a separate test, rather than part of TestParser?

If I use the `EmptyValsFS()` test fixture function to set up the initial test flagset (line 90):

`fs, testVars := fftest.EmptyValsFS()`

...then I think I need to have this test case as a separate test function, `TestParserAllTypesEmptyVals()`. 

I prefer to use this new test fixture function that I created [`EmptyValsFS()`], as it specifies that each type/field in the test field uses specific fields names (e.g.emptyStr, emptyInt, etc.).  I was intentionally creating this new test fixture to help self-document the specifics of this test case in combination with the test function (basically I wanted all the names to match).